CC/Sounder/config.cc

```diff
@@ -217,7 +228,11 @@ Config::Config(const std::string& jsonfile, const std::string& directory,
 #endif
         }
         slot_per_frame_ = calib_frames_.at(0).size();
-        pilot_slot_per_frame_ = 2; // up and down reciprocity pilots
+        if (ref_node_enable_ == true) {
+            pilot_slot_per_frame_ = 2; // Two pilots (up/down)
+        } else {
+            pilot_slot_per_frame_ = 2 * n_bs_sdrs_[0]; // Two pilots per board (up/down)
```

Comment:
I think this should be `num_channels * (n_bs_sdrs_[0] - 1)`

Reply:
I decided to leave it as num_channels * n_bs_sdrs_[0]
we end up with an MxM matrix but I think it's easier to map boards to the matrix during analysis